sdk/python/tests/integration/online_store/test_universal_online.py

```diff
@@ -923,17 +921,17 @@ def test_online_store_cleanup_with_go_server(go_environment, go_data_sources):
     )
 
     fs.apply([driver(), simple_driver_fv, driver_stats_fv])
+    fs._registry.commit()
```

Comment:
is this necessary? `apply` automatically commits at the end

Reply:
I saw race conditions in getting features from the go FS.. It could have been something else tho

Reply:
https://github.com/feast-dev/feast/runs/5484347065?check_suite_focus=true

Example of failure without this line 🤷🏽

Reply:
Isn't this a major red flag? Do we expect users to do this?

Reply:
This is a major blocker for usability. @felixwang9817 and I are debugging this over slack.

Reply:
Okay we've figured this out. The latest version of the code should be green.

The issue was that the feature store class would have left over state in the go server after running `teardown`.